recipes/centrifuge-core/build.sh

```
@@ -1,12 +1,24 @@
+
 export LDFLAGS="-L$PREFIX/lib"
 export CPATH=${PREFIX}/include
 
 mkdir -p $PREFIX/bin
 
+case $(uname -m) in
+    aarch64 | arm64)
+        ARCH_FLAGS="POPCNT_CAPABILITY=0 RELEASE_FLAGS=\"${CXXFLAGS} -fsigned-char\""
+        ;;
+    *)
+        ARCH_FLAGS="RELEASE_FLAGS=\"${CXXFLAGS}\""
```

Shellcheck

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)